### PR TITLE
updates for figma links

### LINKED ISSUE #4534: Issue Template Updates for Figma link
### Current Behavior
The project's current issue templates are missing an open invitation link where new contributors can join the community's Figma team and view user interface designs and other UX projects.

### Desired Situation
Each template that has a reference to Figma in its resources section should an invite link added.

### Implementation 
```
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```

**Acceptance Tests**
All references to Figma include the "open invite" link.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by depleur**
**Description**

This PR fixes #4534

**Notes for Reviewers**
This PR fixes issue template updates for Figma links in certain files.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ghat0tkach** [comment]
@depleur Good work!!
Please sign your commits as well 
Refer [here](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits)

**Comment by l5io** [comment]
🚀 Preview for commit 3de00c54357d957eff3eac9c2a8145523946459d at: https://64b2dc0f29229931320679a3--layer5.netlify.app

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: Retain "Layer5 site". This is the https://layer5.io site repo, not the Meshery UI repo.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by vishalvivekm** [comment]
@depleur 
Please sign off your commits.

Let's discuss it on the websites call.
Add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima), if you would :)

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (depleur) [reply .github/ISSUE_TEMPLATE/bug_report.md:1]: So I'm assuming even the hyperlink needs to be changed to an appropriate Layer5 Figma page?

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
@depleur, incorporate these changes and we're good to go

**Comment by l5io** [comment]
🚀 Preview for commit 2a18fadac153674ff74056ddada3fe991ba7826a at: https://64b7713aad95e249132cb019--layer5.netlify.app

**Comment by HarshDobariya79** [comment]
@depleur your commits are failing DCO checks. Please refer to https://discuss.layer5.io/t/dco-test-in-pr-failed/2836/2 to resolve that.

**Comment by abhijeetgauravm** [comment]
I want to work on it. Can u assign?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ assignees: ''
 
 The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
 - 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
+- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```
[the remark this answers is not in the document]
Comment (Chadha93) [reply .github/ISSUE_TEMPLATE/bug_report.md]: @thisiskaransgit Why Mehsery Figma here? Maybe I'm overlooking...

**Comment by abhijeetgauravm** [comment]
Can I work on this issue?

**Comment by depleur** [comment]
Hey, I've signed this off now so you can look into merging it. Sorry for the delay.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ assignees: ''
 
 The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
 - 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
+ 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/feature_request.md:28]:
```suggestion
 - 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```

Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/feature_request.md:28]:
```suggestion
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```

File: .github/ISSUE_TEMPLATE/chore_task.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ assignees: ''
 
 The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
 - 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
+- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/chore_task.md:19]:
```suggestion
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)

```

**Review by thisiskaransgit — COMMENTED** [review]
@depleur incorporate these, rest all looks good

**Comment by leecalcote** [comment]
@thisiskaransgit thank you for taking care of this PR.

**Pull request merged**